src/storage.c

```diff
@@ -217,3 +217,13 @@ void storage_close(const char *imsi, const char *store, GKeyFile *keyfile,
 
 	g_key_file_free(keyfile);
 }
+
```

Comment:
Please add a short comment here describing the purpose of this function, and why SNAP_COMMON is used.